src/workflows/templates/schemas.ts

```diff
@@ -70,7 +70,7 @@ export function buildCreateRequestSchema(
         type: "object",
         required: ["type"],
         properties: {
-          type: {},
+          type: refs.schemas.types,
```

Comment:
@cmars this is the pattern we've been using, it comes from your common definition

Reply:
Ah, sorry, missed that!

Reply:
I think we want https://raw.githubusercontent.com/snyk/sweater-comb/common-model-v1/components/common.yaml#/schemas/Types -- a `type: string` with a pattern regex.

The linter seems to be failing on an empty type -- updating this might fix it?